src/hooks/useDisableNFTRoutes.ts

```diff
@@ -0,0 +1,6 @@
+import { useAtomValue } from 'jotai/utils'
+import { shouldDisableNFTRoutesAtom } from 'state/application/atoms'
+
+export function useDisableNFTRoutes() {
+  return useAtomValue(shouldDisableNFTRoutesAtom)
+}
```

Comment:
That makes sense, but I changed this to allow for better testing with jest as it was cleaner to just mock `useDisableNFTRoutes()` rather than the entirety of `useAtom`.